### ISSUE TITLE
ub-jsdoc TypeError in the last version when generating documentation

### ISSUE THREAD

**Issue by carla-at-wiris** [UnityBaseJS/ub-jsdoc#11]
I was trying to use the ub-jsdoc template to generate my documentation but i get this error:
```
node_modules/ub-jsdoc/src/parsers.js:66
    if (buildInJSObjects.includes(typeName.toLowerCase())) {

TypeError: Cannot read property 'includes' of undefined
```

with the following configuration:

```
{
    "tags": {
      "allowUnknownTags": true,
      "dictionaries": ["jsdoc"]
    },
    "source": {
      "include": ["src"],
      "includePattern": ".js$",
      "excludePattern": "(node_modules/|docs)"
    },
    "plugins": [
      "plugins/markdown",
      "node_modules/@ckeditor/jsdoc-plugins/lib/export-fixer/export-fixer.js"
    ],
    "templates": {
      "cleverLinks": false,
      "monospaceLinks": true
    },
    "opts": {
      "destination": "./docs/api",
      "encoding": "utf8",
      "private": true,
      "recurse": true,
      "whitelist": ["Optional", "List", "Of", "Top", "Level", "Classes"],
      "title": "",
      "template": "node_modules/ub-jsdoc"
    }
  }
```

I tried to add the following lines to the configuration file:
```
"templates": {
      "cleverLinks": true,
      "monospaceLinks": false,
      "buildInURL": "https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/",
      "buildins": [
        "array",
        "arraybuffer",
        "boolean",
        "date",
        "error",
        "function",
        "json",
        "number",
        "object",
        "regexp",
        "string",
        "null"
      ],
      "smallSourceLink": true,
      "hideAuthor": true,
      "stylesheet": "styles/ub-jsdoc.css",
      "default": {
        "outputSourceFiles": true,
        "staticFiles": {
          "excludePattern": "\\.md$"
        }
      }
    },
```

but then i get this different error from the first one i mentioned:
```
/node_modules/ub-jsdoc/src/parsers.js:106
    return text.replace(/{@link (.*?)}/g, linkReplacer)
                ^

TypeError: Cannot read property 'replace' of undefined
```

I've also looked for projects that are using ub-jsdoc, forked them, actualized the version of ub-jsdoc to the latest and the error still happens on those projects too.

I think maybe it's an issue but if i'm doing something wrong and this causes the error, please let me know so i can use  your template. Thank you

**Comment by RedCarrottt** [comment]
I have same problem with you. Is there any solution?

**Comment by pavelmash** [comment]
If problem is not fixed in 2.5.1 please, provide a link to the sources for which you are generating documentation
